source/release.rst

```diff
@@ -118,16 +118,9 @@ Since we're building with ``noarch``, you can run the below steps on
 either a Linux or macOS machine. You do **not** need to run these steps on both.
 
 
-1. Make sure you have the latest versions of ``anaconda``, ``conda``, and ``conda-build``.
-You cannot be in an existing Anaconda environment when you run ``conda update``,
-so run ``conda deactivate`` first. If the ``conda deactivate`` command doesn't work,
-use ``source deactivate``. This means you have an older version of Anaconda,
-which should be remedied after the following ``update`` command.
-
-    ::
-
-        conda deactivate
-        conda update anaconda conda conda-build
+1. Be in the ``base`` conda environment.
+Make sure you have the latest versions of ``conda``, and ``conda-build``
+by running ``conda update -n base conda conda-build``.
```

Comment:
Not entirely sure this is the correct update to this step.

Reply:
Yes, this is correct.

Reply:
You should add `anaconda-client` to the list.  This is the package that supports the `anaconda upload` command.